apps/lib/win32_init.c

```diff
@@ -211,7 +211,7 @@ void win32_utf8argv(int *argc, char **argv[])
                         p++;
 
                     if (*p == L'"') {
-                        int i;
+                        size_t i;
```

Comment:
Hmm. I'm surprised we don't see warnings about this. Doesn't this `i` shadow the declaration on line 208?